connectors/common-config/src/main/java/com/vmware/connectors/common/config/ConnectorsAutoConfiguration.java

```diff
@@ -5,7 +5,7 @@
 
 package com.vmware.connectors.common.config;
 
-import com.vmware.connectors.common.web.RootController;
+import com.vmware.connectors.common.web.ConnectorRootController;
```

Comment:
There were several classes named "RootController" between APF and connectors, so I renamed them all to be more specific to what they were for and hopefully less confusing.